doc/source/roadmap-detailed.rst

```diff
@@ -355,15 +361,67 @@ functions, and spheroidal wave functions. Three possible ways to handle this:
 stats
 `````
 
-This module is in good shape overall.  New functionality that's similar to
-what's already present can continue to be added; more advanced statistical
-routines may fit better in ``statsmodels``.  Some ideas for new contributions
-are:
-
-- Implementing (well-known) distributions to the ``stats.distributions``
-  framework is always welcome.
-- Continuing work on making the function signatures of ``stats`` and
-  ``stats.mstats`` more consistent, and adding tests to ensure that that
+The ``scipy.stats`` subpackage aims to provide fundamental statistical
+methods as might be covered in standard statistics texts such as Johnson's
+"Miller & Freund's Probability and Statistics for Engineers", Sokal & Rohlf's
+"Biometry", or Zar's "Biostatistical Analysis".  It does not seek to duplicate
+the advanced functionality of downstream packages (e.g. StatsModels,
+LinearModels, PyMC3); instead, it can provide a solid foundation on which
+they can build.  (Note that these are rough guidelines, not strict rules.
+"Advanced" is an ill-defined and subjective term, and "advanced" methods
+may also be included in SciPy, especially if no other widely used and
+well-supported package covers the topic.  Also note that *some* duplication
+with downstream projects is inevitable and not necessarily a bad thing.)
+
+The following improvements will help SciPy better serve this role.
+
+- Add fundamental and widely used hypothesis tests:
+
+  - Alexander-Govern test
+  - Somers' D
+  - Kendall's tau-c
```

Comment:
We already have #9361

Reply:
Right, so it should *definitely* be in the roadmap! :smile:  If the grant comes through, and it turns out that anything we proposed has already been added, that just means we'll have more time to work on additional enhancements.